scipy/stats/_distn_infrastructure.py

```diff
@@ -2212,21 +2239,36 @@ def fit(self, data, *args, **kwds):
               function to be optimized) and ``disp=0`` to suppress
               output as keyword arguments.
 
+            - method : The method to use. The default is "MLE" (Maximum
+              Likelihood Estimate); "GMM" (Generalized Method of Moments)
+              is also available.
+
+            - order : The number of moments to fit with ``method="GMM"``.
+              This integer must be greater than or equal to the number of
+              non-fixed distribution parameters (including location and scale);
+              the default is the number of non-fixed parameters. This keyword
+              is ignored when ``method="MLE"``.
+
         Returns
         -------
-        mle_tuple : tuple of floats
-            MLEs for any shape parameters (if applicable), followed by those
+        parameter_tuple : tuple of floats
+            Estimates for any shape parameters (if applicable), followed by those
             for location and scale. For most random variables, shape statistics
             will be returned, but there are exceptions (e.g. ``norm``).
 
         Notes
         -----
-        This fit is computed by maximizing a log-likelihood function, with
-        penalty applied for samples outside of range of the distribution. The
-        returned answer is not guaranteed to be the globally optimal MLE, it
+        With ``method="MLE"`` (default),
+        the fit is computed by maximizing a log-likelihood function, with
+        penalty applied for samples beyond the range of the distribution.
+
+        With ``method="GMM"``, the fit is computed by minimizing a weighted
+        norm of the errors between data moments and distribution moments.
+
+        The returned answer is not guaranteed to be globally optimal; it
         may only be locally optimal, or the optimization may fail altogether.
-        If the data contain any of np.nan, np.inf, or -np.inf, the fit routine
-        will throw a RuntimeError.
+        If the data contain any of ``np.nan``, ``np.inf``, or ``-np.inf``,
```

Comment:
thanks for fixing these.

Reply:
Haha I see this right after I mention that we're on a timeline. What can I say : P  I was editing the docstring, and these stood out as simple enough.